packages/typography/src/Typography.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import Box, { BoxProps } from '@leafygreen-ui/box';
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+import { useBaseFontSize } from '@leafygreen-ui/leafygreen-provider';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import omit from 'lodash/omit';
+
+const sharedStyles = css`
+  display: block;
```

Comment:
Not everything is display block...code shouldn't be display block. Can we double check these?

Reply:
👍 Fixed!